include/MemoryModel/SymbolTableInfo.h

```diff
@@ -165,6 +163,16 @@ class SymbolTableInfo
     // @{
     static bool isNullPtrSym(const Value *val);
     static bool isBlackholeSym(const Value *val);
+    static bool argInNoCallerFunction(const Value *val);
```

Comment:
Does this mean val is an argument of an uncalled function?

If so, maybe `argOfUncalledFunction` may be clearer.